### PR TITLE
add apex and dwolla processor token support

### PR DESCRIPTION
**Pull request by michaelckelly**
Fixes https://github.com/plaid/plaid-java/issues/86

- Add support for Apex and Dwolla processor tokens

### CODE REVIEW

File: src/main/java/com/plaid/client/request/ApexProcessorTokenCreateRequest.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package com.plaid.client.request;
+
+import com.plaid.client.request.common.BaseAccessTokenRequest;
+
+/**
+ * Request for /item/public_token/exchange endpoint.
+ */
+public final class ApexProcessorTokenCreateRequest extends BaseAccessTokenRequest {
+  private String accessToken;
```
Comment (jeeyoungk) [inline src/main/java/com/plaid/client/request/ApexProcessorTokenCreateRequest.java]: `BaseAccessTokenRequest` already defines `accessToken`, so this line should be removed - it's not used anywhere.

File: src/main/java/com/plaid/client/request/ItemApexProcessorTokenCreateRequest.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import com.plaid.client.request.common.BaseAccessTokenRequest;
+
+/**
+ * Request for  /processor/apex/processor_token/create endpoint.
+ */
+public final class ItemApexProcessorTokenCreateRequest extends BaseAccessTokenRequest {
+  private String accountId;
+
+  public ItemApexProcessorTokenCreateRequest(String accessToken, String accountId) {
+  super(accessToken);
```
Comment (jeeyoungk) [inline src/main/java/com/plaid/client/request/ItemApexProcessorTokenCreateRequest.java]: nit indent

File: src/main/java/com/plaid/client/request/ItemDwollaProcessorTokenCreateRequest.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+import com.plaid.client.request.common.BaseAccessTokenRequest;
+
+/**
+ * Request for  /processor/dwolla/processor_token/create endpoint.
+ */
+public final class ItemDwollaProcessorTokenCreateRequest extends BaseAccessTokenRequest {
+  private String accountId;
+
+  public ItemDwollaProcessorTokenCreateRequest(String accessToken, String accountId) {
+  super(accessToken);
```
Comment (jeeyoungk) [inline src/main/java/com/plaid/client/request/ItemDwollaProcessorTokenCreateRequest.java]: nit indent

**Comment by jeeyoungk** [comment]
Mostly looks good!

**Comment by AndyMoreland** [comment]
Hi! Is there anything I can do to help get this PR merged? It's blocking for my use case right now.

**Comment by emarx** [comment]
Any update @jeeyoungk @michaelckelly? Would love to see this go in!

File: src/main/java/com/plaid/client/request/ItemApexProcessorTokenCreateRequest.java (hunk 2 above)
Comment (michaelckelly) [inline src/main/java/com/plaid/client/request/ItemApexProcessorTokenCreateRequest.java]: :zap:

File: src/main/java/com/plaid/client/request/ApexProcessorTokenCreateRequest.java (hunk 1 above)
Comment (michaelckelly) [inline src/main/java/com/plaid/client/request/ApexProcessorTokenCreateRequest.java]: :zap: this file is actually extraneous, removing

File: src/main/java/com/plaid/client/request/ItemDwollaProcessorTokenCreateRequest.java (hunk 3 above)
Comment (michaelckelly) [inline src/main/java/com/plaid/client/request/ItemDwollaProcessorTokenCreateRequest.java]: :zap:

**Comment by michaelckelly** [comment]
@AndyMoreland  and @emarx - thanks for the ping, looking to get this merged + released to Maven today!

**Comment by michaelckelly** [comment]
@jeeyoungk Your feedback has been addressed, going to go ahead and merge to get this published

**Comment by michaelckelly** [comment]
@AndyMoreland  and @emarx - this change has been released to Maven as `2.1.3`

**Pull request merged**